documentation/milestones/individual_milestones/static_site.md

```diff
@@ -105,12 +103,11 @@ Process for drafting and updating the content of the site should balance:
 5. [Lighthouse score](lighthouse) for the site
 6. Deployment build time
 7. Deployment/hosting costs
-8. Number of PDF views per NOFO prototype
-9. Number of visits to outbound links to the following external resources (once added to the site)
+8. Number of visits to outbound links to the following external resources (once added to the site)
    - Open source repository
    - Open source wiki
    - Open source chat
-10. Stretch: Number of form responses
+9.  Stretch: Number of form responses
```

Comment:
Not asking for a change here but wanted to note a PROTIP: 

> To create an ordered list, add line items with numbers followed by periods. The numbers don’t have to be in numerical order, but the list should start with the number one.

So you can do this: 
```
1. First item
1. Second item
1. Third item
1. Fourth item
```
…and it'll render like this: 
1. First item
1. Second item
1. Third item
1. Fourth item

…then you don't have to worry about setting the right number. 

Aside: This reminds me of [an article](https://siderea.dreamwidth.org/1819759.html) I just read about "a single HTML tag (ordered lists) and end with the downfall of civilization."

Reply:
@andycochran This is a good tip to limit unnecessary diffs!

Currently I use [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) as my VSCode extension for formatting Markdown, which automates the process of updating numbers in ordered lists, so (fortunately) I'm not manually reordering these myself. Still it is a pain to see the diffs on lines that didn't actually result in content changes.

I'll have to check if that VSCode extension supports configuring ordered lists to only use `1.` like you mentioned. Do you use a different markdown extension for VSCode?

Reply:
Also that article is a _wild_ ride, but I love it!! I could feel [the computational law](https://law.stanford.edu/2021/03/10/what-is-computational-law/) recesses of my brain itching as I was reading it 😅

Reply:
Ooh! I'll check out the Markdown All in One extension.

Reply:
_Same_!! Atom was so great, I'm still salty they sunsetted it. And Atom did have a great Markdown plugin that followed the `1.` only style for ordered lists